boards/shields/dfrobot_can_bus_v2_0/dfrobot_can_bus_v2_0.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_GPIO=y
```

Comment:
GPIO shouldn't be needed, as it's part of default board configuration (ie boards should have it enabled by default)

Reply:
Thanks, it has been removed now